index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title> My Portfolio </title>
+</head>
+<body>
+    <header id="header">
+        <div class="head">
+            <h1 id="logo">Fredo</h1>
+            <img class="menu-icon" src="iconMenu.svg" alt="menuIcon">
+        </div>
+        <section class="presentation">
+            <div class="greeting">
+                I’m Fredo Glad to see you!
+            </div>
+            <div class="self-description">
+                I’m a full stack software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="media-connect">
+                <p>LET’S CONNECT</p>
```

Comment:
- [ ] Kindly change this from the `p` tag to the `button` tag or `a` tag. This section is meant to connect you to other sections of the website.